test/tests_for_sdkmr/EE2Runjob_test.py

```diff
@@ -162,7 +168,8 @@ def _set_up_common_return_values(mocks):
     mocks[Workspace].get_object_info3.return_value = {
         "paths": [[_WS_REF_1], [_WS_REF_2]]
     }
-    mocks[Catalog].get_module_version.return_value = {"git_commit_hash": _GIT_COMMIT}
+    mocks[CatalogCache].lookup_git_commit_version.return_value = _GIT_COMMIT
+    mocks[SDKMethodRunner].get_catalog_cache.return_value = mocks[CatalogCache]
```

Comment:
I don't think this line is necessary is it? It's already set up in line 103

Reply:
removed

Reply:
:+1: